components/domains/steps/registerSteps.tsx

```
@@ -6,6 +6,11 @@ import CartIcon from "@/components/UI/iconsComponents/icons/cartIcon";
 import { Skeleton } from "@mui/material";
 import Step from "./step";
 
+type Step = {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'Step'. Consider to delete it or rename it.

'Step' is defined here:


(lint/suspicious/noRedeclare)